tasks/broad/RNAWithUMIsTasks.wdl

```diff
@@ -970,7 +970,7 @@ task PostprocessTranscriptomeForRSEM {
   }
 
   output {
-    File output_bam = "~{prefix}.bam"
+    File output_bam = "~{prefix}_gatk.bam"
```

Comment:
@gbggrant I removed `_gatk` as I thought it wasn't descriptive; did it cause an error without it? Perhaps we should use `_RSEM_post_processed` instead?

Reply:
@takutosato I added it back because that's what you are naming the file on line 969 - I guess you should remove it there in addition.